nixos/modules/system/boot/systemd/initrd.nix

```diff
@@ -364,7 +356,7 @@ in {
 
         "/etc/systemd/system.conf".text = ''
           [Manager]
-          DefaultEnvironment=PATH=/bin:/sbin ${optionalString (isBool cfg.emergencyAccess && cfg.emergencyAccess) "SYSTEMD_SULOGIN_FORCE=1"}
```

Comment:
This is unrelated to this PR, no?

Reply:
Sort of? I needed it because sshd requires the root account to be unlocked.